Livecheckables/ctags.rb

```diff
@@ -0,0 +1,4 @@
+class Ctags
+  livecheck :url   => "http://ctags.sourceforge.net",
+            :regex => /ctags-([0-9\.]+)\.tar\.gz/
```

Comment:
```suggestion
  livecheck :url   => "https://sourceforge.net/projects/ctags/",
            :regex => /ctags-v?(\d+(?:\.\d+)+)\.t/
```

The archive file in the formula is from Sourceforge, so it's preferable for the livecheckable to check the same location.

The `[0-9\.]+` regex here is from old livecheckables and is too loose, since it doesn't adequately express our intention and can match things that we don't want to match. These days, we use some variation on the following regex: `v?(\d+(?:\.\d+)+)`

We also use `\.t` in place of the full extension for tar files, since it could change in the future (e.g., `.tgz`, `.tar.bz2`, `.tar.xz`, etc.). We have to supply _part_ of the extension to effectively restrict matching to stable versions but this is as general as we can get. You'll see `\.t` in a lot of livecheckables.